### PR TITLE
Headrags

### PR DESCRIPTION
**Pull request by Vladdicktorpedo**
Basically this is a 3-in-1 fix. You can now wear certain hairstyles without worrying about the "headrag baldness". 

- Hiding hair flag is disabled for all head-rags. Find your hairstyle for the rags on your own risk - only thing that was done to improve situation is a remake of the "Fade Parted" hairstyle, which was specially developed for wearing the headrags. 

- Long beard sprite was made a little softer. Very long beard now completely wearable. From this moment it's stylish hipster beard.

- Shitspawn-bandana named "infinite ammo" was repainted and now has an authentic look.

### CODE REVIEW

**Comment by Hughgent** [comment]
Would you be so kind as to show a screenshot of it 'in action'? just something from your local computer.

**Comment by Vladdicktorpedo** [comment]
> Would you be so kind as to show a screenshot of it 'in action'? just something from your local computer.

image

**Comment by Vladdicktorpedo** [comment]
its completely done also

File: code/modules/clothing/head/helmet.dm [hunk 1]
```diff
@@ -536,7 +536,7 @@
 	icon_state = "spec"
 	armor = list("melee" = 35, "bullet" = 35, "laser" = 35, "energy" = 15, "bomb" = 10, "bio" = 0, "rad" = 0, "fire" = 15, "acid" = 15)
 	flags_inventory = BLOCKSHARPOBJ
-	flags_inv_hide = HIDEEARS|HIDETOPHAIR
+	flags_inv_hide = HIDEEARS|NONE
```
Comment (psykzz) [inline code/modules/clothing/head/helmet.dm:539]:
```suggestion
	flags_inv_hide = HIDEEARS
```

Comment (psykzz) [reply code/modules/clothing/head/helmet.dm:539]: This is still required

**Pull request merged**